memflow-win32/src/win32/vkey.rs

```diff
@@ -0,0 +1,305 @@
+use core::ops::{Deref, DerefMut};
```

Comment:
What is the purpose of this VKEY implementation? Wouldn't it be better to include a keycodes table in memflow directly and map it to virtual keycodes in memflow-win32 ? This way everything would be more generic and work better with plugins.

Reply:
> What is the purpose of this VKEY implementation? Wouldn't it be better to include a keycodes table in memflow directly and map it to virtual keycodes in memflow-win32 ? This way everything would be more generic and work better with plugins.

Yes, this would be better for *all of memflow* in a generic sense across multiple OS's. These function as the direct rust equivalent of the windows C header VKEY definitions. They are also a transparent integer and get optimized into such. These are copied over from the official windows crate and exported without the extra OS-dependent items mixed in the same module on the windows crate. I also agree that we should have a generic keyboard keys enum, but that these vkey definitions have a place in the windows specific crate as well.